### PR TITLE
TASK: Translate backend messages

### PR DESCRIPTION
**Pull request by pKallert**
**What I did**
In https://github.com/neos/neos-ui/pull/3665 we added the Translation trait to the Backend Service Controller. 
Currently, only one message is translated in the backend, but we should translate all messages.

### CODE REVIEW

File: Resources/Private/Translations/en/Main.xlf [hunk 1]
```diff
@@ -326,6 +326,14 @@
                     <source>Discarded {0} changes.</source>
                 </trans-unit>
             </group>
+            <trans-unit id="switchedBaseWorkspace" xml:space="preserve">
+                <source>Switched base workspace to "{workspace}".</source>
```
Comment (bwaidelich) [inline Resources/Private/Translations/en/Main.xlf:330]: Didn't check and I assume you _did_, but just to make sure: `{workspace}` as placeholder works here? (I only knew that we support numeric placeholders)

**Review by bwaidelich — APPROVED** [review]
+1 by reading

**Review by dlubitz — APPROVED** [review]
👍 Looks also good to me by reading.
(if `{workspace}` really works as placeholder 🤞 )

**Review by crydotsnake — APPROVED** [review]
👁️

File: Resources/Private/Translations/en/Main.xlf (hunk 1 above)
Comment (pKallert) [reply Resources/Private/Translations/en/Main.xlf:330]: Tried it out again to confirm and it looks good: 
Bildschirmfoto 2024-06-14 um 15 26 38

**Pull request merged**